backend/src/main/java/me/performancereservation/api/RefundController.java

```diff
@@ -0,0 +1,72 @@
+package me.performancereservation.api;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import me.performancereservation.domain.refund.RefundService;
+import me.performancereservation.domain.refund.dto.RefundDetailResponse;
+import me.performancereservation.domain.refund.dto.RefundRequest;
+import me.performancereservation.domain.refund.enums.RefundStatus;
+import me.performancereservation.global.exception.AppException;
+import me.performancereservation.global.exception.ErrorCode;
+import me.performancereservation.global.exception.ErrorType;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/refunds")
+public class RefundController {
+
+    private final RefundService refundService;
+
+    /*--- USER 요청에 대응 ---*/
+    // 본인 id와 일치하는 모든 환불내역 리스트 반환
+    @GetMapping("/user/{userId}")
+    public ResponseEntity<List<RefundDetailResponse>> getAllRefundDetailsWithUserId(@PathVariable Long userId) {
+        return ResponseEntity.status(HttpStatus.OK)
+                .body(refundService.findAllRefundsDetailByUserId(userId));
+    }
+
+    // 특정 예약id 환불요청 -> 환불내역 생성
+    @PostMapping("/user/new-refund")
+    public ResponseEntity<Long> createNewRefund(@RequestBody RefundRequest refundRequest) {
+        return ResponseEntity.status(HttpStatus.CREATED)
+                .body(refundService.save(refundRequest));
+    }
+
+
+    /*--- ADMIN 요청에 대응 ---*/
+    // 모든 환불 리스트 반환
+    @GetMapping("/admin/all-refunds")
```

Comment:
```suggestion
    @GetMapping("/admin")
```

이부분도 /refunds 자체가 리소스를 표현한것이기 때문에 all-refunds는 중복으로 표현한거라 제거해주시는게 좋을 것 같아요

뭔가 admin/refunds였으면 뭔가 더 나았으려나? 싶기도 한데 컨트롤러단에서 refunds를 글로벌 프리픽스로 달았기 때문에 이정도는 제 생각엔 괜찮은 것 같습니다

Reply:
반영하였습니다!